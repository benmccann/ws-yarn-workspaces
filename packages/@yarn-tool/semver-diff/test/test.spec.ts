import { parseVersionsDiffCore } from '..';

test(`parseVersionsDiffCore`, () =>
{

	let actual = parseVersionsDiffCore('42.6.7.9.3-alpha', '42.6.7.9.3-prealpha');
	let expected;

	//expect(actual).toStrictEqual(expected);
	expect(actual).toMatchSnapshot();

});

