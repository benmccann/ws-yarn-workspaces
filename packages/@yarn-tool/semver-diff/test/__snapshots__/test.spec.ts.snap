// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseVersionsDiffCore 1`] = `
Object {
  "_colors": Array [
    "red",
    "cyan",
    "green",
  ],
  "color": "green",
  "comp": 1,
  "index": 2,
  "leadingWildcard": "",
  "middot": ".",
  "partsNew": Array [
    "42",
    "6",
    "7.9.3-prealpha",
  ],
  "partsOld": Array [
    "42",
    "6",
    "7.9.3-alpha",
  ],
  "result": "42.6",
  "resultAppend": "7.9.3-prealpha",
  "versionNew": "42.6.7.9.3-prealpha",
  "versionOld": "42.6.7.9.3-alpha",
}
`;
